<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StringDialog</class>
 <widget class="QDialog" name="StringDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>81</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>String</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="toolTip">
        <string>The text.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonFileDialog">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Open a file dialog.</string>
       </property>
       <property name="icon">
        <iconset resource="../ressources.qrc">
         <normaloff>:/icons/icons/open127.png</normaloff>:/icons/icons/open127.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>&amp;Ok</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>StringDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>StringDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
